src/AuthorCard/AuthorCard.vue

```diff
@@ -0,0 +1,45 @@
+<template>
+  <Card>
+    <template #header>
+      <img src="profilePicture" alt="John" style="width: 100%" />
+    </template>
```

Comment:
You need to use the Avatar component created by Harshal to handle this

Reply:
after you deploy, I will use his component.